main/src/main/resources/graph.html

```diff
@@ -0,0 +1,115 @@
+<!doctype html>
+
+<!--
+
+Based on https://github.com/cpettitt/dagre-d3/blob/d215446e7e40ebfca303f4733e746e96420e3b46/demo/interactive-demo.html
+which is published under this license:
+
+Copyright (c) 2013 Chris Pettitt
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+-->
+
+<meta charset="utf-8">
+<title>Dependency Graph</title>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+<script src="https://www.samsarin.com/project/graphlib-dot/v0.6.1/graphlib-dot.js"></script>
```

Comment:
Maybe this a good time to stop using this samsarin.com domain...

Reply:
Sure, I added a commit. Unfortunately `graphlib-dot` is not actively maintained, so the best I could find, according to [its own wiki](https://github.com/dagrejs/graphlib-dot/wiki#browser-scripts) was to use GH cdn. Version `0.6.1` was not available despite being marked as the latest release in the [release page](https://github.com/dagrejs/graphlib-dot/releases) but version `0.6.3` which seems to be the latest version was. I tested the HTML page after this upgrade and it works fine.

There are other alternatives (now in 2020) so maybe in long term switching to other l`dot` libraries is a good move.